flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/KubernetesOperatorConfigOptions.java

```diff
@@ -143,6 +143,18 @@ public class KubernetesOperatorConfigOptions {
                     .withDescription(
                             "Whether to enable recovery of missing/deleted jobmanager deployments. False by default for Flink 1.14, true for newer Flink version.");
 
+    public static final ConfigOption<Integer> OPERATOR_SAVEPOINT_HISTORY_MAX_COUNT =
+            ConfigOptions.key("kubernetes.operator.savepoint.history.max.count")
+                    .intType()
+                    .defaultValue(10)
+                    .withDescription("Maximum number of savepoint history entries to retain.");
+
+    public static final ConfigOption<Duration> OPERATOR_SAVEPOINT_HISTORY_MAX_AGE =
```

Comment:
Since we are performing a lazy clean-up, the savepoint may live longer than the max age.

Reply:
I added this to the doc.